README.md

```diff
@@ -16,14 +16,16 @@ interface as the Enterprise Edition has a more comprehensive filtering engine.
 Please consult the chapter ["Integrating Checkmk in Grafana"][3] in the
 official Checkmk documentation on how to use this plugin.
 
-![Checkmk Grafana Data Source Plugin](../assets/grafana-checkmk-datasource.png)
+![Checkmk Grafana Data Source Plugin](https://github.com/tribe29/grafana-checkmk-datasource/raw/assets/grafana-checkmk-datasource.png)
```

Comment:
Can we get a permalink here?

It should be this one:
https://github.com/tribe29/grafana-checkmk-datasource/raw/ebf24142922ccce5cc5649aa4809d1c19d55958f/grafana-checkmk-datasource.png

Reply:
:+1: